WDAC-Policy-Wizard/app/src/ConfigTemplate_Control.cs

```diff
@@ -505,23 +505,49 @@ private bool ReadSetRules(object sender, EventArgs e)
             }
             else
             {
-                switch (this.Policy._PolicyTemplate)
+                if(this.Policy._Format == WDAC_Policy.Format.MultiPolicy)
                 {
-                    case WDAC_Policy.NewPolicyTemplate.WindowsWorks:
-                        // Windows Works Mode 
-                        xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "DefaultWindows_Audit.xml");
-                        break;
-
-                    case WDAC_Policy.NewPolicyTemplate.SignedReputable:
-                        // Signed and Reputable Mode
-                        xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "SignedReputable.xml");
-                        break;
-
-                    case WDAC_Policy.NewPolicyTemplate.AllowMicrosoft:
-                        // Allow Microsoft mode
-                        xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "AllowMicrosoft.xml");
-                        break;
+                    // Multi-policy Format Policy Templates
+                    switch (this.Policy._PolicyTemplate)
+                    {
+                        case WDAC_Policy.NewPolicyTemplate.WindowsWorks:
```

Comment:
formatting nitpick, `case` should be at same indentation level as brace

Reply:
Woah, never knew that. Thanks!